packages/react-integration/cypress/integration/toolbar.spec.ts

```diff
@@ -5,6 +5,13 @@ describe('Data Toolbar Demo Test', () => {
     cy.url().should('eq', 'http://localhost:3000/toolbar-demo-nav-link');
   });
 
+  it('has inset classes', () => {
```

Comment:
can you test all breakpoints.  We have had issues in the past when classes where not being applied correctly at certain breakpoints.  See Tabs for an example.